apps/election-manager/src/components/ExportElectionBallotPackageModalButton.tsx

```diff
@@ -0,0 +1,301 @@
+            </Button>
+          )
+          mainContent = (
+            <Prose>
+              <h1>USB Drive Detected!</h1>
+              <p>
+                <USBImage src="usb-drive.svg" alt="Insert USB Image" />
+                Would you like to export the ballot configuration now? A zip
+                archive will automatically be saved to the inserted USB drive.
+                You may also {/* eslint-disable jsx-a11y/anchor-is-valid */}
+                <a href="#" onClick={openFileDialog} data-testid="manual-link">
+                  manually select a location to save the archive to.
+                </a>
+                {/* eslint-enable jsx-a11y/anchor-is-valid */}
+              </p>
+            </Prose>
+          )
+          break
+        }
+      }
+      break
+
+    case 'RenderBallot': {
+      disableCancel = true
+      const {
+        ballotStyleId,
+        precinctId,
+        contestIds,
+        isLiveMode,
+        locales,
+      } = state.currentBallotConfig
+      const precinctName = getPrecinctById({ election, precinctId })!.name
+
+      mainContent = (
+        <Prose>
+          <h1>
+            Generating Ballot{' '}
+            {state.ballotConfigsCount - state.remainingBallotConfigs.length} of{' '}
+            {state.ballotConfigsCount}…
+          </h1>
+          <ul>
+            <li>
+              Ballot Style: <strong>{ballotStyleId}</strong>
+            </li>
+            <li>
+              Precinct: <strong>{precinctName}</strong>
+            </li>
+            <li>
+              Contest count: <strong>{contestIds.length}</strong>
+            </li>
+            <li>
+              Language format:{' '}
+              <strong>{getHumanBallotLanguageFormat(locales)}</strong>
+            </li>
+            <li>
+              Filename:{' '}
+              <Monospace>{state.currentBallotConfig.filename}</Monospace>
+            </li>
+          </ul>
+          <HandMarkedPaperBallot
+            ballotStyleId={ballotStyleId}
+            election={election}
+            electionHash={electionHash}
+            isLiveMode={isLiveMode}
+            precinctId={precinctId}
+            onRendered={onRendered}
+            locales={locales}
+          />
+        </Prose>
+      )
+      break
+    }
+
+    case 'ArchiveEnd': {
+      mainContent = (
+        <Prose>
+          <h1>Finishing Download…</h1>
+          <p>
+            Rendered {pluralize('ballot', state.ballotConfigsCount, true)},
+            closing zip file.
+          </p>
+        </Prose>
+      )
+      break
+    }
+
+    case 'Done': {
+      primaryButton = <USBControllerButton primary small={false} />
+      mainContent = (
+        <Prose>
+          <h1>Download Complete!</h1>
+          <p>
+            Exported {pluralize('ballot', state.ballotConfigsCount, true)}. You
+            may now eject the USB device and connect it with your ballot
+            scanning machine to configure it.
+          </p>
```

Comment:
Suggestion to simplify this modal. Number of ballots exported is not something which 
```suggestion
          <h1>Export Complete</h1>
          <p>
            You may now eject the USB drive. Use the exported ballot package on this USB drive to configure the Ballot Scanner.
          </p>
```

Reply:
Addressed in https://github.com/votingworks/vxsuite/pull/97/commits/98fd424c5290ff406ebe5302330c8bdbb381072a 

New Screen 
<img width="613" alt="Screen Shot 2020-12-08 at 2 48 38 PM" src="https://user-images.githubusercontent.com/14897017/101551997-e5d01980-3966-11eb-9c6a-2c9391cf99f2.png">

Reply:
Please remove the exclamation point in the modal title.